curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e55b4b06c11fff555c64.md

```diff
@@ -19,28 +19,28 @@ Your `calculateCalories` function should have a `surplusOrDeficit` variable.
 assert.match(calculateCalories.toString(), /surplusOrDeficit\s*=/g);
 ```
 
-You should have a ternary operator that checks if `remainingCalories` is greater than or equal to `0`.
+You should have a ternary operator that checks if `remainingCalories` is less than `0`.
```

Comment:
tests description and tests were also updated to reflect the new ternary operator